specification/network/resource-manager/Microsoft.Network/stable/2022-11-01/examples/ApplicationGatewayCreate.json

```diff
@@ -507,52 +436,7 @@
               }
             }
           ],
-          "urlPathMaps": [
-            {
-              "name": "pathMap1",
-              "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/applicationGateways/appgw/urlPathMaps/pathMap1",
-              "properties": {
-                "provisioningState": "Succeeded",
-                "defaultBackendAddressPool": {
-                  "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/applicationGateways/appgw/backendAddressPools/appgwpool"
-                },
-                "defaultBackendHttpSettings": {
-                  "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/applicationGateways/appgw/backendHttpSettingsCollection/appgwbhs"
-                },
-                "defaultRewriteRuleSet": {
-                  "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/applicationGateways/appgw/rewriteRuleSets/rewriteRuleSet1"
-                },
-                "defaultLoadDistributionPolicy": {
-                  "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/applicationGateways/appgw/loadDistributionPolicies/ldp1"
-                },
-                "pathRules": [
-                  {
-                    "name": "apiPaths",
-                    "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/applicationGateways/appgw/urlPathMaps/pathMap1/pathRules/apiPaths",
-                    "properties": {
-                      "provisioningState": "Succeeded",
-                      "paths": [
-                        "/api",
-                        "/v1/api"
-                      ],
-                      "backendAddressPool": {
-                        "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/applicationGateways/appgw/backendAddressPools/appgwpool"
-                      },
-                      "backendHttpSettings": {
-                        "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/applicationGateways/appgw/backendHttpSettingsCollection/appgwbhs"
-                      },
-                      "rewriteRuleSet": {
-                        "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/applicationGateways/appgw/rewriteRuleSets/rewriteRuleSet1"
-                      },
-                      "loadDistributionPolicy": {
-                        "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/applicationGateways/appgw/loadDistributionPolicies/ldp1"
-                      }
-                    }
-                  }
-                ]
-              }
-            }
-          ],
+          "urlPathMaps": [],
```

Comment:
we need to remove the same from other example json such as Get and List, ListAll, e.g.,: https://github.com/Azure/azure-rest-api-specs/blob/59b0c933be875d3d064aefb172230a5d008419ed/specification/network/resource-manager/Microsoft.Network/stable/2022-11-01/examples/ApplicationGatewayGet.json

Reply:
Actually, the response includes empty resources. For example: 
![image](https://user-images.githubusercontent.com/127259533/225981468-eaf7ec09-5c0b-4a7f-9d83-86237434b9e9.png)